packages/oui-dual-list/src/dual-list.html

```diff
@@ -0,0 +1,89 @@
+<div class="oui-dual-list oui-dual-list___full_height">
+    <div class="oui-dual-list__source">
+        <div class="oui-dual-list__header">
+            <span>{{$ctrl.sourceListLabel}}</span>
+            <span data-ng-if="!$ctrl.sourceListLoading">({{$ctrl.sourceList.length}})</span>
+            <span class="oui-dual-list__header_toggle">
+                <i class="oui-icon oui-icon-chevron-up oui-dual-list__header_toggle_up"
+                    data-ng-style="$ctrl.sourceOpenIconStyle"
+                    data-ng-click="$ctrl.onSourceContentOpen()"
+                    aria-hidden="true"></i>
+                <i class="oui-icon oui-icon-chevron-down oui-dual-list__header_toggle_down"
+                    data-ng-style="$ctrl.sourceCloseIconStyle"
+                    data-ng-click="$ctrl.onSourceContentClose()"
+                    aria-hidden="true"></i>
+            </span>
+        </div>
+        <div class="oui-dual-list__content oui-dual-list___full_height" data-ng-style="$ctrl.sourceContentStyle">
+            <div class="oui-dual-list__empty_label" data-ng-if="!$ctrl.sourceListLoading && !$ctrl.sourceList.length">
+                <p>{{$ctrl.sourceListEmptyLabel}}</p>
+            </div>
+            <div class="oui-dual-list__empty_label" data-ng-if="$ctrl.sourceListLoading">
+                <oui-spinner size="m"></oui-spinner>
+            </div>
+            <div class="oui-dual-list___full_height" data-ng-if="!$ctrl.sourceListLoading && $ctrl.sourceList.length">
+                <div class="oui-dual-list__content_filter">
+                    <input type="text" data-ng-model="$ctrl.sourceFilterKey" ng-model-options="{debounce: 500}"/>
+                    <i class="oui-icon oui-icon-search" aria-hidden="true"></i>
+                </div>
+                <div class="oui-dual-list__content_add_all" data-ng-if="$ctrl.bulkActionEnabled">
+                    <a data-ng-click="$ctrl.addAll()">{{$ctrl.moveAllLabel}}</a>
+                </div>
+                <ul class="oui-dual-list__content_list" data-ng-style="$ctrl.sourceListHeight">
+                    <li data-ng-repeat="source in $ctrl.sourceList | ouiDualListFilter:$ctrl.sourceFilterKey:$ctrl.property"
+                        data-ng-click="$ctrl.add($index, source)"
+                        class="oui-dual-list__content_list_item">
+                        <span data-ng-if="!$ctrl.isLoading(source)" class="oui-dual-list__content_list_item_icon">
```

Comment:
For accessibility purpose, we should not use `span`.
We can't navigate with the keyboard in the component.

Prefer using `button type="button"` with `ngClick` on it, instead on `li`